layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -10,6 +10,8 @@
 ;;; License: GPLv3
 
 (require 'cl-lib)
+(eval-when-compile
+  (require 'tramp))
```

Comment:
Move this into the function that uses it, maybe? That way we don't load tramp on startup. (I know tramp is usually loaded on startup anyway, but…)